src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/AsyncTest/AsyncTest.cs

```diff
@@ -1,106 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Diagnostics;
-using System.Threading.Tasks;
-using Xunit;
-
-namespace Microsoft.Data.SqlClient.ManualTesting.Tests
-{
-    public static class AsyncTest
-    {
-        [ActiveIssue("5533")] // Async Operations slower than Sync operations
-        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
-        public static void TestReadAsyncTimeConsumed()
```

Comment:
I think this is a regression test - its purpose is to make sure that async isn't inherently slower than sync. Is this functionality tested elsewhere?

Reply:
Assert.True(elapsedAsync < elapsedSync, "Asynchronous operation should be finished quicker than synchronous one");

It really wants the async code to be faster than the sync code.

Also, it has been disabled for at least 5 years now....    :)

Reply:
That's a fair point - no objections to removing it!